### ISSUE TITLE
Find conversion script

### ISSUE THREAD

**Issue by edreamleo** [leo-editor/leo-editor#2318]
This script converted .leo files with old sentinels. I may have posted it to leoeditor.com.

Indeed yes. Searching for 'convert leo files' in leoeditor .com yields [this page](https://groups.google.com/g/leo-editor/search?q=convert%20leo%20files).

#1581: Allow Leo to read ancient .leo files contains a script in the [second comment](https://github.com/leo-editor/leo-editor/issues/1581#issuecomment-619179616), newly unhidden.

Hah!  leoImport.py contains the LegacyExternalFileImporter class, and Leo supports the import-legacy-external-files command.  Presumably this is the polished version of the script above.

**Comment by edreamleo** [comment]
I am going to close this issue. Please reopen if questions remain.

**Comment by tbpassin** [comment]
Reopened just for this comment.  I found these external files in the Leo Git tree that have the version 4 format (I'm omitting ones that are obviously not going to matter, like in unittest):

```
leo-editor\leo\extensions\testExtension.py
leo-editor\leo\extensions\colors.py
leo-editor\leo\external\stringlist.py
leo-editor\leo\scripts\leoFindScript.py
```
BTW, for those who don't know it, I found these files using FileLocator Pro.  This Windows only program can find files in any subtree of your drive, with the names or extensions wildcarded, containing any text you want to find.  It's amazingly fast, doesn't spend time indexing your drive, and has a free-for-home use version.
